300_Aggregations/55_approx_intro.asciidoc

```diff
@@ -60,5 +60,18 @@ approximations--but the vast majority will have no problem.  Accepting
 an approximate answer is more often a _cultural_ hurdle rather than a business
 or technical imperative.
 
+In case you are still seeking very high levels of approximation, and are willing to
+potentially use much more resources to get the answer, this is a summary of the 
+steps you can take:
+
+- For `cardinality`, you can configure `precision_threshold` to make the algorithm 
+exact almost 100% of cases, for values with a cardinality lower than `precision_threshold`.
+- For `percentiles`, error rates are less predictable/generalizable, but you can 
```

Comment:
I'm not a fan of this phrasing, since percentiles has bounded error just like cardinality (it's just a different shape).  

How about:  

>For `percentiles`/`percentile_ranks`, approximation error can be controlled by setting the `compression` parameter. The default percentiles algorithm -- TDigest -- approximates extreme percentiles (1%, 99%) more accurately than the interior percentiles, which tends to be more useful in analytics anyway.  See LINK for more details

Also link to the docs about the compression settings